en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -334,6 +334,103 @@ When there are null values in custom expressions and are:
+Each variable must specify the `Name` and data type. To set the properties for the variable definition, complete the following:
+... `Variable Label (Title)`:  The display label for the variable. A value is required. 
+... `Type` : The data type of the variable. The following xref:ms_composer_custom_expression_editor.adoc#data-types[data types] are supported: `String`, `Number`, and `Boolean`. To define a `Date` or `DateTime` variable, the data type should be set to `String` and the `Format` property should be defined. A value is required. 
+... `Format`:  The qualifier for `String` properties that can specify additional data types not natively supported in JSON. For variables, this must be set when defining `Date` or `DateTime` variables as `date` or `date-time` respectively. A value in this field is optional. 
+
+//this should be a result statement... but how?! The Dev insisted that we have this section but I have no idea how to format it and I really don't even know how to introduce it.
+The properties are entered into the *JSON Schema* window. For example: 
+
+[source,JSON]
+----
+{
+"properties": {
+"var-01": {
+"type": "string",
+"title": "My String"
+},
+"var-02": {
+"type": "number",
+"title": "My Number"
+},
+"var-03": {
+"type": "boolean",
+"title": "My Boolean"
+},
+"var-04": {
+"type": "string",
+"title": "My Date",
+"format": "date"
+},
+"var-05": {
+"type": "string",
+"title": "My Datetime",
+"format": "date-time"
+}
+}
+}
+----
+//Does anyone have any idea how to indent this block and make the step numbers continue instead of restart?! 
+. Optionally, to add additional variables in the JSON Schema, enter them in the window.
+. Click *Apply*. +
+The *Create Variables* window appears, displaying one or more new variables added to the flow. 
+. Optionally, to make changes to the new variables, click *Manage Variables* to open the Visual Editor or *Edit Schema* to open the JSON Schema.
+. Complete the initial value for each variable. The default value is defined under each variable. If the field is empty, the value is set to the default value for that data type. You can enter a value manually, select any existing data pill (including variables) from your flow, or you can select the xref:ms_composer_custom_expression_editor.adoc#add-custom-expression-to-an-action[*Custom Expression Editor*] to create a custom expression. 
+
+=== Update a Variable 
+
+To update one or more variables: 
+
+. <<access-the-variable-step,Access the *Variable* step.>> 
+. In the *Action* dropdown, click *Update Variable*.
+. In the *Variable* section, click *Select Variables*. +
+The *Select Variables* window appears, displaying a list of all variables included in your flow before this step. 
```

Comment:
Necessary?

Reply:
Ashley was very specific about this one. She wants it known that it ONLY shows variables that were added prior to this step and nothing after.